__tests__/components/ResourceCreation/TestResourceCreation.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { fireEvent, render, screen } from '@testing-library/react';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
```

Comment:
```suggestion
import { fireEvent, render, screen } from '@testing-library/react';
```
We can get rid of these imports since they are no longer being used in this file